lib/src/base-styles/_nx-alert.scss

```diff
@@ -82,3 +82,7 @@
     color: #007623;
   }
 }
+
+.nx-page-main > .nx-alert {
+  margin-bottom: 24px;
```

Comment:
I see that alerts have a default top/bottom margin of 8px.  Where does that value come from? It seems awfully small and I don't see it anywhere in the mockups. I wonder if it's a holdover from 0.x and if we should just set the default top/bottom margin to 24px

Reply:
> I see that alerts have a default top/bottom margin of 8px. Where does that value come from? It seems awfully small and I don't see it anywhere in the mockups. I wonder if it's a holdover from 0.x and if we should just set the default top/bottom margin to 24px

I went ahead and applied our standard `16px` above, `24px` below component margin to the alert.https://github.com/sonatype/sonatype-react-shared-components/pull/227/commits/d41d9571814c91fbd73b9138e6107347264675fa

Reply:
> > I see that alerts have a default top/bottom margin of 8px. Where does that value come from? It seems awfully small and I don't see it anywhere in the mockups. I wonder if it's a holdover from 0.x and if we should just set the default top/bottom margin to 24px
> 
> I went ahead and applied our standard `16px` above, `24px` below component margin to the alert.[d41d957](https://github.com/sonatype/sonatype-react-shared-components/commit/d41d9571814c91fbd73b9138e6107347264675fa)

@dturner32 ^^